build/manifest/main.go

```diff
@@ -145,7 +146,10 @@ func findManifest() (*model.Manifest, error) {
 
 	// Update the release notes url to point at the latest tag, if present.
 	if BuildTagLatest != "" {
-		manifest.ReleaseNotesURL = manifest.HomepageURL + "releases/tag/" + BuildTagLatest
+		manifest.ReleaseNotesURL, err = url.JoinPath(manifest.HomepageURL, "releases/tag", BuildTagLatest)
```

Comment:
I'm not 100% sure, but maybe `"release/tag"` can be `"release", "tag", and that way it would work even in windows (I don't think is needed, but there is no harm in it.

Reply:
Thanks, @jespino! Can you expand on the Windows perspective to help me understand -- this is a URL, so was thinking it should always be `/`, but you're right that we could just have `url.JoinPath` "do that" instead of manually composing the two.

Reply:
Totally right, this is a URL, so my comments makes no sense.